pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux_test.go

```diff
@@ -162,7 +162,7 @@ func TestManager(t *testing.T) {
 			overrideSystemInhibitDelay:       time.Duration(40 * time.Second),
 			enablePodDisruptionConditions:    true,
 			expectedDidOverrideInhibitDelay:  false,
-			expectedPodToGracePeriodOverride: map[string]int64{"running-pod": 20, "failed-pod": 20, "succeeded-pod": 20},
+			expectedPodToGracePeriodOverride: map[string]int64{"running-pod": 30, "failed-pod": 30, "succeeded-pod": 30},
```

Comment:
A nit but could you define this somewhere and use that throughout the tests?  

If you could use the constant you defined in the other file that would be ideal.